src/web/components/FluentUIProvider.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
```

Comment:
Do we really need this component? could we add this provider in a root component?

Reply:
No, it's something weird with vscode that highlighted I needed it at a certain point. But shouldn't be needed